apps/modernization-ui/src/apps/page-builder/pages/ValuesetLibrary/ValuesetLibraryTable.tsx

```diff
@@ -284,20 +295,20 @@ export const ValuesetLibraryTable = ({ summaries, labModalRef, pages }: Props) =
             <div>{<SearchBar onChange={setSearchQuery} />}</div>
             {summaries?.length ? (
                 <TableComponent
-                    contextName="valuesets"
                     tableHeader=""
                     tableHead={tableColumns}
                     tableBody={tableRows}
                     isPagination={true}
-                    pageSize={pages?.pageSize || 0}
-                    totalResults={pages?.totalElements || 0}
-                    currentPage={pages?.current || 0}
+                    pageSize={pages?.pageSize}
+                    totalResults={pages?.totalElements}
+                    currentPage={pages?.currentPage}
                     handleNext={setCurrentPage}
                     sortData={handleSort}
                     selectable
                     handleSelected={handleSelected}
                     rangeSelector={true}
                     isLoading={isLoading}
+                    context={useContext(ValueSetsContext)}
```

Comment:
We should use createContext and useContext and not need to pass through props:
https://react.dev/learn/passing-data-deeply-with-context